src/java.base/share/classes/java/text/MessageFormat.java

```diff
@@ -110,17 +110,21 @@
  * </pre></blockquote>
  *
  * <p>
- * The <i>ArgumentIndex</i> value is a non-negative integer written
+ * The <b>ArgumentIndex</b> value is a non-negative integer written
```

Comment:
If we would deviate from the format in the above form, I'd use @code tag, instead of explicit <b>.